kylin/src/main/java/org/apache/zeppelin/kylin/KylinInterpreter.java

```diff
@@ -148,6 +146,22 @@ public HttpResponse prepareRequest(String sql) throws IOException {
     return httpClient.execute(postRequest);
   }
 
+  public String getProject(String cmd) {
```

Comment:
Hi, this kind of style is used by JDBCInterpreter, but I think it's too tricky and removed from JDBCInterpreter for now. And we started to discuss on how to support this kind of property - that we called as note level properties -. I don't think your implementation makes a big problem but it can confuse existing users.